features/support/journey_helpers.rb

```diff
@@ -61,12 +74,27 @@ def select_random_upper_tier_options(carrier)
 end
 
 def select_random_lower_tier_options
+  return answer_random_lower_tier_questions unless @resource_object == :new_registration
+
+  i = rand(2)
+  if i == 0
+    @journey.check_your_tier_page.submit(option: :unknown)
+
+    answer_random_lower_tier_questions
+
+    @journey.standard_page.submit
+  else
+    @journey.check_your_tier_page.submit(option: :lower)
+  end
+end
+
+def answer_random_lower_tier_questions
   # Randomise between 3 ways to achieve an lower tier registration:
```

Comment:
```suggestion
  # Randomise between 4 ways to achieve an lower tier registration:
```

Reply:
This is still 3, why do you think is now 4? 🤔

Reply:
This is still 3 ways

Reply:
Sorry, I wrote this before properly understanding the code structure. I had thought that the 4th way was saying "I need a lower tier registration". But actually, this choice is made somewhere else.